1-js/02-first-steps/05-types/article.md

```diff
@@ -216,9 +216,9 @@ alert(x); // "undefined"
 
 نوع `object` (شیء) از انواع خاص است.
 
-انواع مختلف داده که تا اینجا مطالعه کردیم از انواع «اولیه» بودند چرا که مقدار آنها فقط شامل یک چیز می‌شد (رشته یا عدد یا هر چیز دیگری). در مقابل، شیءها برای ذخیره مجموعه‌ای از داده‌ها به شکلی پیچیده‌تر استفاده می‌شوند.
+انواع مختلف داده که تا اینجا مطالعه کردیم از انواع «primitive» بودند چرا که مقدار آنها فقط شامل یک چیز می‌شد (رشته یا عدد یا هر چیز دیگری). در مقابل، شیءها برای ذخیره مجموعه‌ای از داده‌ها به شکلی پیچیده‌تر استفاده می‌شوند.
 
-شیءها به حدی مهم هستند که لایق طرز برخورد ویژه‌ای هستند. ما بعدا در فصل <info:object> به سراغ آنها خواهیم رفت، بعد از اینکه درباره انواع اولیه بیشتر یاد گرفتیم.
+شیءها به حدی مهم هستند که لایق طرز برخورد ویژه‌ای هستند. ما بعدا در فصل <info:object> به سراغ آنها خواهیم رفت، بعد از اینکه درباره انواع primitive بیشتر یاد گرفتیم.
```

Comment:
I'm not sure about reverting the translation of "primitive".
What's your opinion, @mahdyar?

Reply:
> I'm not sure about reverting the translation of "primitive". What's your opinion, @mahdyar?

Maybe adding the Persian equivalent in parentheses be a better idea? 🤔

Reply:
> Maybe adding the Persian equivalent in parentheses be a better idea? 🤔

Alright, then it'll be reverted.